internal/resources/resource_volume.go

```diff
@@ -256,11 +262,18 @@ func resourceMetalVolumeRead(d *schema.ResourceData, meta interface{}) (err erro
 	loc, _ := p.GetLocationName(volume.LocationID)
 	d.Set(vLocation, loc)
 	d.Set(vLocationID, volume.LocationID)
+
 	if err = d.Set(vShareable, volume.Shareable); err != nil {
 		return err
 	}
+
 	d.Set(vState, volume.State)
 	d.Set(vStatus, volume.Status)
+	err = d.Set(vWWN, volume.WWN)
+
+	if err != nil {
+		return fmt.Errorf("set WWN: %v", err)
+	}
```

Comment:
```suggestion
	if err = d.Set(vWWN, volume.WWN); err != nil {
		return fmt.Errorf("set WWN: %v", err)
	}
```

Reply:
That is what I have before, but the lint failed and this is the only way it is taking.....
internal/resources/resource_volume.go:272:2: if statements should only be cuddled with assignments (wsl)
        if err = d.Set(vWWN, volume.WWN); err != nil {
        ^
make: *** [Makefile:67: go-lint] Error 1